src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.html

```diff
@@ -2,7 +2,13 @@ <h1 mat-dialog-title>GitHub App Logs</h1>
 <div mat-dialog-content>
   <app-loading [loading]="loading">
     <mat-card *ngIf="showContent === 'error'" class="alert alert-warning mat-elevation-z" role="alert">
-      <mat-icon class="alert-warning-icon">warning</mat-icon> There were problems retrieving GitHub App logs for this organization.
+      <mat-icon class="alert-warning-icon">warning</mat-icon> There were problems retrieving the GitHub App logs for this organization.<br />
+      Typically, if you cannot view the logs for your GitHub user account, you must unlink and relink it in<br />
+      your Dockstore profile.<br />
```

Comment:
You could optionally link to the Dockstore accounts page.

Reply:
Done.